<launch>


  <!-- Launching USB CAM launch files and Dynamixel controllers -->
  <include file="$(find eyecu)/launch/sensors/start_usb_cam.launch"/>

  <!-- Launch velodyne lidar -->
  <include file="$(find eyecu)/launch/sensors/VLP16_points.launch"/>

  <!-- Launch transforms -->
  <include file="$(find eyecu)/launch/static_transforms.launch.xml"/>

  <!-- Starting face tracker node -->
   <rosparam file="$(find eyecu)/config/track.yaml" command="load"/>
   <node name="eyecu" pkg="eyecu" type="eyecu_node_dlib" output="screen"/>

   <node name="laser_image" pkg="eyecu" type="laser_image_projection" output="screen"/>

</launch>
